rest/route/host.go

```diff
@@ -351,7 +352,22 @@ func (ch offboardUserHandler) Factory() gimlet.RouteHandler {
 }
 
 func (ch *offboardUserHandler) Parse(ctx context.Context, r *http.Request) error {
-	ch.user = gimlet.GetVars(r)["user_id"]
+	input := struct {
```

Comment:
Did the mana people request it be done like this? It seems really finicky with how people have a bunch of email addresses that redirect to the same mailbox. Can we validate that the user name we parsed exists in the db?

Reply:
Yeah they did request it :/ it's how they want all of mana 2 to work, they just want to send the same payload to every route. Makes sense to validate the user exists, I'll add that.